index.html

```diff
@@ -55,6 +55,9 @@
     {% assign filename = filename | replace: ".", "-dot-" %}
     {% assign filename = filename | replace: "&", "-and-" %}
     {% assign filename = filename | replace: " ", "" | replace: "!", "" | replace: ":", "" | replace: "’", "" | replace: "'", "" | replace: "°", "" %}
+    {% if icon.slug %}
```

Comment:
Should this not be an `if/else` statement (assuming Liquid supports them), to avoid all those replacements if `slug` _is_ present?

Reply:
Liquid does support `if/else` statements and we can write it using one, this is just the lazy version (from the programmer's perspective): Liquid will first do all the replacements on the filename - for all icons - and if the icon has a slug specified then the filename is **replaced** by the slug. It's just wasting a bunch of CPU cycles for icons that have a slug.

I'd be happy to change it if you prefer :slightly_smiling_face:

Reply:
Actually, given that we'll have switched over to the new site before we have enough `slug` entries for this to be a major issue, I don't think it's such a big deal if we don't update it.

Reply:
That was my reasoning for this going with this easier but more obscure solution as well :smile: